### PR TITLE
EVENT-644: currency symbol based on user locale 

### PR DESCRIPTION
**Pull request by katefree**

### CODE REVIEW

File: app/views/eventDetails/regTypes.html [hunk 1]
```diff
@@ -280,8 +280,12 @@ <h4 translate>Payment</h4>
                 <label translate>Minimum Payment to Register</label>
                 <div class="input-group">
                   <span class="input-group-addon"
-                    ><i class="fa fa-usd"></i
-                  ></span>
+                    ><i class="fa"
```
Comment (OzzieOrca) [inline app/views/eventDetails/regTypes.html]: Is this still using font awesome? If the `fa` style is exactly what you want, you can keep it. Just seems a little odd to have font awesome html and then not use font awesome for the icon.

File: app/views/eventDetails/regTypes.html [hunk 2]
```diff
@@ -280,8 +280,12 @@ <h4 translate>Payment</h4>
...
+                      ><span
+                        >&nbsp;{{conference.currency.shortSymbol}}&nbsp;</span
```
Comment (OzzieOrca) [inline app/views/eventDetails/regTypes.html]: You could probably replace `&nbsp;` with padding.

File: app/views/eventDetails/regTypes.html [hunk 3]
```diff
@@ -356,8 +360,12 @@ <h4 translate>Early Registration Discount(s)</h4>
                   <label translate>Discount</label>
                   <div class="input-group">
                     <span class="input-group-addon"
-                      ><i class="fa fa-usd"></i
-                    ></span>
+                      ><i class="fa"
+                        ><span
+                          >&nbsp;{{conference.currency.shortSymbol}}&nbsp;</span
```
Comment (OzzieOrca) [inline app/views/eventDetails/regTypes.html]: Same here.

File: app/views/modals/choiceOptions.html [hunk 4]
```diff
@@ -31,7 +31,7 @@ <h3 translate>Options</h3>
     <label translate>Additional Cost</label>
     <div class="input-group">
       <span class="input-group-addon"
-        ><span>&nbsp;{{conference.currency.shortSymbol}}&nbsp;</span></span
+        ><span>&nbsp;{{currency}}&nbsp;</span></span
```
Comment (OzzieOrca) [inline app/views/modals/choiceOptions.html]: Same here.

File: app/views/modals/paymentsModal.html [hunk 5]
```diff
@@ -386,8 +386,12 @@ <h4>
                       >
                         <div class="input-group">
                           <span class="input-group-addon"
-                            ><i class="fa fa-usd"></i
-                          ></span>
+                            ><i class="fa"
+                              ><span
+                                >&nbsp;{{conference.currency.shortSymbol}}&nbsp;</span
```
Comment (OzzieOrca) [inline app/views/modals/paymentsModal.html]: Same here.

File: app/views/eventDetails/regTypes.html (hunk 1 above)
Comment (katefree) [reply app/views/eventDetails/regTypes.html]: I have tried just to remove it and it become italic. These fa class unfortunately exists also in many other places so maybe it worth refactor it in the separate pull request, please let me know what you think

File: app/views/eventDetails/regTypes.html (hunk 2 above)
Comment (katefree) [reply app/views/eventDetails/regTypes.html]: fixed

File: app/views/eventDetails/regTypes.html (hunk 3 above)
Comment (katefree) [reply app/views/eventDetails/regTypes.html]: fixed

File: app/views/modals/choiceOptions.html (hunk 4 above)
Comment (katefree) [reply app/views/modals/choiceOptions.html]: fixed

File: app/views/modals/paymentsModal.html (hunk 5 above)
Comment (katefree) [reply app/views/modals/paymentsModal.html]: fixed

File: app/views/eventDetails/regTypes.html (hunk 1 above)
Comment (OzzieOrca) [reply app/views/eventDetails/regTypes.html]: Nothing's forcing you to use the `<i>` tag. It's just the element font awesome decided to use for their icons.

**Pull request merged**